### DISCUSSION TITLE
Marketplace Service degradation

### DISCUSSION THREAD

**Discussion by prashantvc** [microsoft/vsmarketplace#380] [Announcement]
We are experiencing the service degradation for marketplace or unavailability. We are working on the fix, we appreciate your patience

https://status.dev.azure.com/

Refresh the marketplace page or run search multiple times in VS Code if it fails as a workaround

**Comment by radeksimko** [comment]
@prashantvc Thanks for the heads up!

The service seems pretty slow from the UK as well but the status page reports all regions other than the US as green - is that correct?

**Comment by prashantvc** [comment]
Things should be back to normal! We have restored the marketplace services
You can find more details about the outage here https://status.dev.azure.com/_event/302592133
